src/Translate/TranslateClient.php

```diff
@@ -31,19 +31,18 @@
  * information at
  * [Google Translate docs](https://cloud.google.com/translate/docs/).
  *
- * Please note that unlike most other Cloud Platform services Google Translate
- * requires a public API access key and cannot currently be accessed with a
- * service account or application default credentials. Follow the
+ * Please note that while Google Translate supports authentication via service
```

Comment:
Need to update the content here as well: https://github.com/GoogleCloudPlatform/google-cloud-php/blob/master/src/ServiceBuilder.php#L271-L276